### ISSUE TITLE
add verbose output when a ssl error occurs

### ISSUE THREAD

**Issue by lepapareil** [Orange-OpenSource/hurl#145]
Using hurl to call an https server whose certificates had expired I found that the error output was not accurate enough, example:

```shell
test.hurl: failure
error: Http Connection
   --> test.hurl:16:5
    |
 16 | GET https://localhost/test
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ SSL certificate problem
```

It would be interesting to have the complete error stack allowing us to understand that it was the cause of a certificate expiry in first reading.

**Comment by lepapareil** [comment]
Using hurl to call an https server whose certificates had expired I found that the error output was not accurate enough, example:

```shell
test.hurl: failure
error: Http Connection
   --> test.hurl:16:5
    |
 16 | GET https://localhost/test
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ SSL certificate problem
```

It would be interesting to have the complete error stack allowing us to understand that it was the cause of a certificate expiry in first reading.

**Comment by fabricereix** [comment]
implemented in #148
